en/docs/deploy-and-publish/publish-on-dev-portal/publish-aws-apis-in-the-dev-portal.md

```diff
@@ -0,0 +1,118 @@
+# Importing APIs from AWS API Gateway to the WSO2 API Manager  
+
+This feature helps publish an API a user has in the AWS API gateway, on the WSO2 developer portal. These APIs will not be deployed into the WSO2 API gateway and will not be managed by the WSO2 API gateway.
+
+## Prerequisites 
+
+1. Two WSO2 API Controller commands are needed for this task. Hence the user needs to have the APICTL downloaded and configured. The apictl is a command line interface similar to the AWS CLI. [Getting started with the API Controller](https://apim.docs.wso2.com/en/latest/learn/api-controller/getting-started-with-wso2-api-controller/)
+
+2. The `aws init` command will be using the AWS CLI in the background. Hence the user needs to download and install the AWS CLI before using the `aws init` command. Refer [how to download and install the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html).
+
+3. After successfully installing the AWS CLI it needs to be configured. Refer [how to configure the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html).
+
+4. Enable AWS CLI pagination. By default the AWS CLI returns all objects in the output, but if you have disabled pagination you will have to enable pagination for the `aws init` command to successfully work. 
+For more information, refer [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-pagination.html).
+
+5. The API Manager server needs to be up and running to import the API project to the APIM successfully.
+
+6. An active internet connection is also required to download the OAS of the AWS API to initialize the API project.
+
+## Importing an API from AWS API gateway to the WSO2 API Manager  
+
+Usage of the `aws init` command is quite simple. Two mandatory flags are associated with the `aws init` command, `name` and `stage`. `name` will be the name of the API that is in the AWS API gateway that the user wants to import and publish in the developer portal and `stage` will be the specific stage of that API.
+
+-   **Command**
+
+    ``` bash
+    apictl aws init -n <aws-api-name> -s <api-stage-to-get>
+    ```
+
+!!! example
+
+    ```bash
+    apictl aws init -n PetStore -s demo
+    ```
+
+!!! info
+    **Flags:**  
+            
+    -    Required :  
+
+        `--name` or `-n` : Name of the API to get from the AWS API Gateway
+
+        `--stage` or `-s` : API stage to get 
+
+    -   Optional :
+
+        `-f` : To force create an API project
+
+        `--verbose` or `-v` : To execute the command in verbose mode
```

Comment:
```suggestion
        `--name` or `-n` : Name of the API to get from the AWS API Gateway
        `--stage` or `-s` : API stage to get 
    -   Optional :
        `-f` : To force create an API project
        `--verbose` or `-v` : To execute the command in verbose mode
```

Reply:
Let's remove all the extra lines here. (You can refer to other API Controller doc pages to get an idea of how we have mentioned Required and Optional flags of apictl commands)